paasta_tools/utils.py

```diff
@@ -1466,6 +1467,16 @@ def get_cluster(self) -> str:
     def get_dashboard_links(self) -> Dict[str, Dict[str, str]]:
         return self.config_dict['dashboard_links']
 
+    def get_auto_hostname_unique_size(self) -> int:
+        """
+        We automatically add a ["hostname", "UNIQUE"] constraint to "small" services running in production clusters.
+        If there are less than or equal to this number of instances, we consider it small.
+        We fail safe and return 0 to avoid adding the ['hostname', 'UNIQUE'] constraint if this value is not defined
+
+        :returns: The integer size of a small service
+        """
+        return self.config_dict.get('auto_hostname_unique_size', 0)
```

Comment:
This needs to be -1 actually, because otherwise when someone "disables" a service it has to bounce to get a new constraint. I would rather it just scale from 0-1 without a bounce.